downstream/modules/aap-hardening/con-deployment-methods.adoc

```diff
@@ -13,4 +13,4 @@ This document further recommends using the container-based installation method f
 
 For further information, see link:{URLReleaseNotes}/aap-2.5-deprecated-features#aap-2.5-deprecated-features[Deprecated features].
 
-Operator-based deployments are out of scope for this document.
\ No newline at end of file
+Operator-based deployments are not covered in this document.
```

Comment:
This is a tiny nit, maybe worthless, but I wonder if "covered" is one of those odd English words that is harder to translate. How do you feel about "included" or "reviewed" instead? Please feel free to disregard this idea.